core/templates/dev/head/css/oppia.css

```diff
@@ -2246,6 +2246,7 @@ md-card.oppia-dashboard-tile {
   background-color: #009688;
   border-radius: 5px;
   color: white;
+  display: inline-block;
   font-size: 10px;
   margin-left: 3px;
```

Comment:
Yes, without the margin-left it looks good both behind the exploration title or on its own line. Updated PR with changes.